frontend/src/components/DashboardTable.vue

```diff
@@ -1,77 +1,108 @@
-      <v-row class="px-4">
-        <v-col md="5">
-          <div v-if="unreadExchangeCount > 0">
-            <v-badge bordered top color="rgb(0, 51, 102)" circle offset-x="23" offset-y="30">
-              <v-icon aria-hidden="false" color="rgb(0, 51, 102)" size="100">
-                mdi-email-outline
-              </v-icon>
-            </v-badge>
-          </div>
-          <div v-else >
-            <v-icon aria-hidden="false" color="rgb(0, 51, 102)" size="100">
-              mdi-email-outline
-            </v-icon>
-          </div>
+  <v-row class="d-flex justify-center">
+    <v-col md="5" sm="10">
+      <v-row no-gutters>
+        <v-col cols="6">
+          <v-card class="mt-0 mb-5" width="22em"  outlined rounded @click="redirectToInbox()">
+            <v-row class="pl-4">
+              <v-col cols="4">
+                <div v-if="unreadExchangeCount > 0">
+                  <v-badge bordered top color="rgb(0, 51, 102)" circle offset-x="23" offset-y="30">
+                    <v-icon aria-hidden="false" color="rgb(0, 51, 102)" size="100">
+                      mdi-email-outline
+                    </v-icon>
+                  </v-badge>
+                </div>
+                <div v-else >
+                  <v-icon aria-hidden="false" color="rgb(0, 51, 102)" size="100">
+                    mdi-email-outline
+                  </v-icon>
+                </div>
+              </v-col>
+              <v-col class="mt-2">
+                <v-card-title class="pa-0">
+                  <h4>
+                    <v-row class="dashboard-title mr-4">{{ title }}</v-row>
+                  </h4>
+                </v-card-title>
+                <v-row><span> {{exchangeCount}} messages, {{unreadExchangeCount}} unread</span></v-row>
+              </v-col>
+            </v-row>
+          </v-card>
         </v-col>
-        <v-col class="mt-4 mr-1">
-          <v-card-title class="pa-0">
-            <h4>
-              <v-row class="dashboard-title mr-4">{{ title }}</v-row>
-            </h4>
-          </v-card-title>
-          <v-row class="mr-4"><span> {{exchangeCount}} messages, {{unreadExchangeCount}} unread</span></v-row>
+        <v-col cols="6">
+          <v-card v-if="isLoggedInDistrictUser" width="22em"  class="mt-0 mb-5" outlined rounded @click="redirectToDistrictDetails()">
+            <v-row class="pl-4">
+              <v-col cols="4">
+                <div>
+                  <v-icon aria-hidden="false" color="rgb(0, 51, 102)" size="100">
+                    mdi-domain
+                  </v-icon>
+                </div>
+              </v-col>
+              <v-col class="mt-2">
+                <v-card-title class="pa-0">
+                  <h4>
+                    <v-row class="dashboard-title mr-4">{{ PAGE_TITLES.DISTRICT_DETAILS }}</v-row>
+                  </h4>
+                </v-card-title>
+                <v-row><span> Last updated {{schoolsLastUpdateDate}}</span></v-row>
```

Comment:
should this be districtLastUpdateDate?

Reply:
@sukanya-rath - All the "lastUpdated" logic is not working properly so I left it for now. We need to fix them all.